jaxrs-spec/src/main/asciidoc/chapters/introduction/_status.adoc

```diff
@@ -28,3 +28,11 @@ The project team seeks feedback from the community on any aspect of this
 specification, please send comments to:
 
 jaxrs-dev@eclipse.org
+
+[[context-injection]]
+==== Support for @Context Injection
```

Comment:
I am no adoc expert, so maybe this is dumb question: Is it sufficient / typical to *only* have the equals lines to the left?